### PR TITLE
[3.5] chore(e2e): backport TestNoErrorLogsDuringNormalOperations test

### PR DESCRIPTION
**Pull request by ghouscht**
Backport of https://github.com/etcd-io/etcd/pull/18806 to `release-3.5` branch.

Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Review by serathius — APPROVED** [review]

**Comment by ivanvc** [comment]
/retitle [3.5] chore(e2e): backport TestNoErrorLogsDuringNormalOperations test
/ok-to-test

**Review by fuweid — APPROVED** [review]

File: tests/e2e/logging_test.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+func TestNoErrorLogsDuringNormalOperations(t *testing.T) {
+	tests := []struct {
+		name          string
+		clusterSize   int
+		allowedErrors map[string]bool
+	}{
+		{
+			name:          "single node cluster",
+			clusterSize:   1,
+			allowedErrors: map[string]bool{},
```
Comment (ahrtr) [inline tests/e2e/logging_test.go:36]: I do not get time to dig out why 3.5 doesn't see the error message as 3.6. Did you ever dig into it?

**Review by ahrtr — APPROVED** [review]
Thank you!

File: tests/e2e/logging_test.go (hunk 1 above)
Comment (ghouscht) [reply tests/e2e/logging_test.go:36]: 3.5: https://github.com/ghouscht/etcd/blob/release-3.5/server/embed/etcd.go#L882-L892

main: https://github.com/ghouscht/etcd/blob/main/server/embed/etcd.go#L872-L885

Added with https://github.com/etcd-io/etcd/pull/14116

Comment (ahrtr) [reply tests/e2e/logging_test.go:36]: Thanks.

**Pull request merged**